cluster/gce/manifests/etcd.manifest

```diff
@@ -24,6 +24,7 @@
     "command": [
               "/bin/sh",
               "-c",
+              # If etcd name defined by --name is modified, please make sure ETCD_NAME in migrate-if-needed.sh is changed to the same: https://github.com/kubernetes/kubernetes/blob/v1.19.0-beta.2/cluster/images/etcd/migrate-if-needed.sh#L49.
```

Comment:
This doesn't seem to work.
You may check if moving the comment outside of the "command" (i.e put it before the
`"comment": [`
line would help

Reply:
Ah you are right, I added "comment" instead and rebuilt k8s and tried kube-up, it works now.